### DISCUSSION TITLE
Possibility to choose what part of script to be run

### DISCUSSION THREAD

**Discussion by stefankinell** [karttapullautin/karttapullautin#6] [Ideas]
Would it be possible to add parameters to the .ini-file that can steer what part of the script that will be executed.

e.g. - run off cliffs and contours and only do vegetation. 

Rational:
New vegetation is possible in single file mode, but not in batch mode. This will help a lot if you want to redo the vegetation only with new parameters.

**Comment by rphlo** [comment]
Vegetation only mode has been added in latest versions

**Comment by rphlo** [comment]
Since version 2.2.0 we added the possibility to run:
- Vegetation Only
- Contours Only
- Cliffs Only
